src/Scripts/Game/Features/ID/Licence.c

```diff
@@ -0,0 +1,15 @@
+class Licence
+{
+	protected RTTextureWidget m_wRenderTargetTextureWidget;
+	TextWidget licenceTextWidget;
+	
+	void Licence(string widgetname, string displayname, Widget m_wPIPRoot)
+	{
+		m_wRenderTargetTextureWidget = RTTextureWidget.Cast(m_wPIPRoot.FindAnyWidget("RTTexture0"));
```

Comment:
```suggestion
		m_wRenderTargetTextureWidget = RTTextureWidget.Cast(wPIPRoot.FindAnyWidget("RTTexture0"));
```

Reply:
Same here as it will be taking from the methods given variables.